### PR TITLE
Kundemodul

### PR DESCRIPTION
**Pull request by Craksy**
add utility methods for `TECHCOOL` and tweak data model

### CODE REVIEW

File: ErpSystemOpgave/ErpSystemOpgave/Data/Address.cs [hunk 1]
```diff
@@ -1,7 +1,8 @@
-using System;
```
Comment (AlienNotFound) [inline ErpSystemOpgave/ErpSystemOpgave/Data/Address.cs:1]: what - we talked about this

File: ErpSystemOpgave/ErpSystemOpgave/Data/Person.cs [hunk 2]
```diff
@@ -1,22 +1,41 @@
-using System;
```
Comment (AlienNotFound) [inline ErpSystemOpgave/ErpSystemOpgave/Data/Person.cs:1]: silas..

File: ErpSystemOpgave/ErpSystemOpgave/Data/SalesOrderLine.cs [hunk 3]
```diff
@@ -1,6 +1,3 @@
-﻿using System;
```
Comment (AlienNotFound) [inline ErpSystemOpgave/ErpSystemOpgave/Data/SalesOrderLine.cs:1]: fine.

**Review by AlienNotFound — CHANGES_REQUESTED** [review]
yes. but pls add using systems again.

**Comment by Craksy** [comment]
pff, okay... be like that

File: ErpSystemOpgave/ErpSystemOpgave/Data/Address.cs (hunk 1 above)
Comment (Craksy) [reply ErpSystemOpgave/ErpSystemOpgave/Data/Address.cs:1]: The address class does not use the `System` namespace

File: ErpSystemOpgave/ErpSystemOpgave/Data/Person.cs (hunk 2 above)
Comment (Craksy) [reply ErpSystemOpgave/ErpSystemOpgave/Data/Person.cs:1]: The person class doesn't either

**Comment by Craksy** [comment]
> yes. but pls add using systems again.

Looked again, and the `System` name space weren't beeing used in any of the removed instances.

so... stop being such a cunt about it?

**Review by AlienNotFound — APPROVED** [review]
Fine you knob, I'll just approve it then.

**Pull request merged**